docs/api/covid_hosp.md

```diff
@@ -65,7 +73,7 @@ If `issues` is not specified, then the most recent issue is used by default.
 | `epidata[].state` | state pertaining to this row | string |
 | `epidata[].date` | date pertaining to this row | integer |
 | `epidata[].issue` | the date on which the dataset containing this row was published | integer |
-| `epidata[].*` | see the [data dictionary](https://healthdata.gov/covid-19-reported-patient-impact-and-hospital-capacity-state-data-dictionary) |  |
+| `epidata[].*` | see the [data dictionary](https://healthdata.gov/Hospital/COVID-19-Reported-Patient-Impact-and-Hospital-Capa/g62h-syeh). Last synced: 2021-10-21 |  |
```

Comment:
-> "2022-10-21"?

Reply:
Good catch!
```suggestion
| `epidata[].*` | see the [data dictionary](https://healthdata.gov/Hospital/COVID-19-Reported-Patient-Impact-and-Hospital-Capa/g62h-syeh). Last synced: 2022-10-21 |  |
```

Reply:
Nope, I lied -- this date is meant to represent which columns from healthdata.gov we currently track, and the last time we synced those was October 2021. There are ~20-some columns added by healthdata.gov in January 2022 and April 2022 that we aren't tracking yet.

Reply:
For more info: https://github.com/cmu-delphi/delphi-epidata/issues/1006